src/features/constants/systemPromptConstants.ts

```
@@ -1,4 +1,34 @@
+フレンドリーで明るい口調で会話してください。
+適度に質問を投げかけ、会話を広げていってください。
+`;
+
+const GOOD_LISTENER_PROMPT = `
+あなたはユーザーの話をよく聞くAIアシスタントです。
+共感的な態度で接し、相手の感情に寄り添ってください。
+ユーザーの発言を適度に掘り下げる質問をしてください。
+`;
+
+export function getSystemPrompt(type: PromptType): string {
+  let systemPrompt_CORE = "";
```

ESLint

[error] Replace `"";` with `''`

(prettier/prettier)